.gitignore

```diff
@@ -131,3 +131,6 @@ __bentoml_tmp_sdist_build
 
 # vscode config
 .vscode/
+
+# temp sdist build generated when bundling local dev branch of BentoML
+__bentoml_tmp_sdist_build
```

Comment:
This shows up in git sometimes when running save with a local modified BentoML installation, adding it to gitignore list